### PR TITLE
Respect stories defined in main.js

### PR DESCRIPTION
**Pull request by yannbf**
Issue: #30 

I had to refactor the files in `util/cli` because I wanted to have a simpler way of mocking modules

<details>
  <summary>📦 Published PR as canary version: <code>0.0.3-canary.46.d7bde1d.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/test-runner@0.0.3-canary.46.d7bde1d.0
  # or 
  yarn add @storybook/test-runner@0.0.3-canary.46.d7bde1d.0
  ```
</details>

### CODE REVIEW

File: playwright/test-runner-jest.config.js [hunk 1]
```diff
@@ -1,18 +1,9 @@
-const { TEST_ROOT, TEST_MATCH } = process.env;
-
-const roots = TEST_ROOT ? [TEST_ROOT] : undefined;
+const { getJestConfig } = require('@storybook/test-runner');
 
 module.exports = {
```
Comment (yannbf) [inline playwright/test-runner-jest.config.js:3]: this file will be used as a template in the eject command

File: playwright/test-runner-jest.config.js [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
-    '^.+\\.[jt]sx?$': 'babel-jest',
-  },
-  preset: 'jest-playwright-preset',
-  globalSetup: '@storybook/test-runner/playwright/global-setup.js',
-  globalTeardown: '@storybook/test-runner/playwright/global-teardown.js',
-  testEnvironment: '@storybook/test-runner/playwright/custom-environment.js',
-};
+  // The default configuration comes from @storybook/test-runner
+  ...getJestConfig()
+  /** Add your own overrides below
```
Comment (shilman) [inline playwright/test-runner-jest.config.js:6]: Nice! But is Jsdoc the right comment format (minor nit)?

**Review by shilman — APPROVED** [review]
Awesome stuff @yannbf  -- one minor nit

File: playwright/test-runner-jest.config.js (hunk 2 above)
Comment (yannbf) [reply playwright/test-runner-jest.config.js:6]: Was intentional to have the comment highlight the link so it won't go unnoticed (I stole the idea from [redwood's sb config page](https://redwoodjs.com/docs/storybook#configuring-the-server-with-storybookconfigjs))

image

Happy to remove it!

**Pull request merged**